packages/dashboard/src/vaadin-dashboard-widget.js

```diff
@@ -68,10 +68,19 @@ import { DashboardSection } from './vaadin-dashboard-section.js';
  *
  * The following custom properties are available:
  *
- * Custom Property                   | Description
- * ----------------------------------|-------------
- * `--vaadin-dashboard-item-colspan` | colspan of the widget
- * `--vaadin-dashboard-item-rowspan` | rowspan of the widget
+ * Custom Property                             | Description
+ * --------------------------------------------|----------------------
+ * `--vaadin-dashboard-item-colspan`           | colspan of the widget
+ * `--vaadin-dashboard-item-rowspan`           | rowspan of the widget
+ * `--vaadin-dashboard-widget-background`      | widget background
+ * `--vaadin-dashboard-widget-border-radius`   | widget corner radius
+ * `--vaadin-dashboard-widget-border-width`    | widget border width
+ * `--vaadin-dashboard-widget-border-color`    | widget border color
+ * `--vaadin-dashboard-widget-shadow`          | widget shadow (non edit mode)
+ * `--vaadin-dashboard-widget-editable-shadow` | widget edit mode shadow
+ * `--vaadin-dashboard-widget-selected-shadow` | selected widget shadow
```

Comment:
Are we sure we want to introduce state-based properties? I’m not sure all themes would specifically adjust the shadow for the editable and selected states. Feels a bit arbitrary to have state-specific props just for the shadow. And I don't like the idea of introducing state-based versions of all the props (background, border, etc).